### ISSUE TITLE
Field type timestamp mysql default value

### ISSUE THREAD

**Issue by sj8m** [facebookincubator/ent#558]
Is it possible to add default current timestamp and on update current timestamp to the mysql migrate schema? I tried using it like below but it throws an error. This is the error when migrating:
```
2020/06/18 13:28:07 failed creating schema resources: sql/schema: mysql: unsupported type: time.Time
```
![Screenshot 2020-06-18 at 15 40 43](https://user-images.githubusercontent.com/53516395/85027334-17b3b080-b17a-11ea-9ab1-4042d470403b.png)

**Comment by a8m** [comment]
For time fields, ent doesn't set the default value on the database. However, in cases like this, we can just ignore it. Please paste here your schema and I'll send a patch for this.

**Comment by sj8m** [comment]
Yeah the schema's of this project are private 😅 So I can't send the schema. Is it possible to fix this in a future version? The code above is a mixin for the create_time and update_time. I need the default and on update for a client who wants to insert records from Microsoft access and want to have create_time and update_time to be set by default :)

**Comment by a8m** [comment]
I didn't mind the whole schema, just these 2 fields in the screenshot 😄 .

**Comment by a8m** [comment]
I didn't mean the whole schema, just these 2 fields in the screenshot 😄.

**Comment by sj8m** [comment]
The screenshot from above is enough?

**Comment by sj8m** [comment]
Do you need something else from me?

**Comment by a8m** [comment]
Hey @sj8m and sorry for the delay.
I've landed 3e7481c3a015400fd5f407aba1c57f35731601dc that ignores the MySQL default values for time fields, and it should not apply any changes for the second time the migration runs.

However, it won't work with the current way you declare the schema, because you add additional parameters to the schema datatype (the create/update default values).

I suggest to close this issue and open a new one for adding support for fields/edges dialect-specific DDLs. For example:

```go
// Add abstraction for common DDLs.
field.Time("create_time").
    DDL(
        field.Default("CURRENT_TIMESTAMP"),
        field.OnUpdate("CURRENT_TIMESTAMP")
    )

// Add additional information for the given column. The EXTRA in the
// INFORMATION_SCHEMA table. For example, the "ON UPDATE" clause.
field.Time("update_time").
    SchemaDef(map[string]string{
        MySQL: "DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP",
    })

// Completely override the ent-migration column definition.   
field.Time("create_time").
    SchemaDef(map[string]string{
        MySQL: "TIMESTAMP DEFAULT CURRENT_TIMESTAMP NOT NULL",
    })
```
